src/main/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImpl.java

```diff
@@ -0,0 +1,184 @@
+package edu.cornell.kfs.sys.service.impl;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.amazonaws.ClientConfiguration;
+import com.amazonaws.PredefinedClientConfigurations;
+import com.amazonaws.SdkClientException;
+import com.amazonaws.services.secretsmanager.AWSSecretsManager;
+import com.amazonaws.services.secretsmanager.AWSSecretsManagerClientBuilder;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueRequest;
+import com.amazonaws.services.secretsmanager.model.GetSecretValueResult;
+import com.amazonaws.services.secretsmanager.model.UpdateSecretRequest;
+import com.amazonaws.services.secretsmanager.model.UpdateSecretResult;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.service.AwsSecretService;
+
+public class AwsSecretServiceImpl  implements AwsSecretService{
```

Comment:
A space should be added before the curly brace. There also seems to be an extra unneeded space prior to the "implements" keyword.

Reply:
ok